shared/cfg/guest-os/Linux/Ubuntu/16.04-server.s390x.cfg

```diff
@@ -0,0 +1,13 @@
+- 16.04-server.s390x:
+    image_name += -16.04-server-s390x
+    vm_arch_name = s390x
+    os_variant = ubuntutrusty
+    unattended_install, svirt_install:
+        kernel = images/ubuntu-server-16-04-s390/vmlinuz
+        initrd = images/ubuntu-server-16-04-s390/initrd.gz
+        boot_path = install
+    unattended_install.cdrom, svirt_install:
+        unattended_file = unattended/Ubuntu-16-04.preseed
```

Comment:
Not found unattended/Ubuntu-16-04.preseed file,  it looks you missed this file in your patch.

Thanks,
Xu